### PR TITLE
Bug 1868394: Fix boot-order change alert when VM is running

### PR DESCRIPTION
**Pull request by irosenzw**
Show the alert when the disk order is changing in the yaml
while the VM is running.

Signed-off-by: Ido Rosenzwig <irosenzw@redhat.com>

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/selectors/vm-like/next-run-changes.ts [hunk 1]
```diff
@@ -84,7 +84,9 @@ export const isBootOrderChanged = (vm: VMWrapper, vmi: VMIWrapper): boolean => {
     const vmDevices = getDevices(vm.asResource());
     const vmiDevices = getVMIDevices(vmi.asResource());
 
-    return vmDevices.every((bootableDevice, index) => _.isEqual(bootableDevice, vmiDevices[index]));
+    return !vmDevices.every((bootableDevice, index) =>
```
Comment (glekner) [inline frontend/packages/kubevirt-plugin/src/selectors/vm-like/next-run-changes.ts:87]: why not
 `vmDevices.some((bootableDevice, index) => _.isEqual(bootableDevice, vmiDevices[index])`

Comment (irosenzw) [reply frontend/packages/kubevirt-plugin/src/selectors/vm-like/next-run-changes.ts:87]: you mean:
`vmDevices.some((bootableDevice, index) => !_.isEqual(bootableDevice, vmiDevices[index])` ?

**Pull request merged**